support-frontend/app/controllers/Application.scala

```diff
@@ -220,7 +221,7 @@ class Application(
     Ok(
       views.html.main(
         title = "Guardian Supporters Map",
-        mainElement = assets.getSsrCacheContentsAsHtml("aus-moment-map", "aus-moment-map.html"),
+        mainElement = EmptyDiv("aus-moment-map"),
```

Comment:
[This doesn't actually exist](https://github.com/guardian/support-frontend/blob/main/support-frontend/assets/helpers/rendering/ssrPages.ts#L1-L9) and is another example of [having quite flexible fallbacks which make it quite hard to work out when something is or isn't doing](https://github.com/guardian/support-frontend/blob/7676de2bca6110564c9d86a4559a375ce5a3b10a/support-frontend/app/assets/AssetsResolver.scala#L62).